docs/operators/s3outputmodule.md

```diff
@@ -0,0 +1,149 @@
+S3OutputModule
+==============
+
+### About Amazon S3
+-------------------
+
+**Amazon S3 (Simple Storage Service)** is a web service interface that you can use to store and retrieve any amount of data at any time from anywhere on the web offered by Amazon Web Services.
```

Comment:
is a web service interface => is an object storage system with a web service interface
on the web offered => on the web, offered